content/videos/challenges/3-snake-game/showcase/contribution11.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Python Edition",
+  "url": "",
```

Comment:
Curious what this looks like on the site, might want to put the GitHub link here instead

Reply:
You're right, definitely need to fill in that url or else it redirects to the same page in another tab :)